authorization_services/topics/permission-create-resource.adoc

```diff
@@ -3,7 +3,12 @@
 
 A resource-based permission defines a set of one or more resources to protect using a set of one or more authorization policies.
 
+ifeval::[{project_community}==true]
+To create a new resource-based permission, select *Create resource-based permission* from the *Create permission* dropdown.
+endif::[]
+ifeval::[{project_product}==true]
 To create a new resource-based permission, select *Resource-based* in the item list in the upper right corner of the permission listing.
+endif::[]
 
```

Comment:
"Apply Policy" on line 40 needs to be updated. Now I see "Policy" instead. I don't know if it is simply a name change.